Dockerfile

```diff
@@ -1,14 +1,34 @@
-FROM debian
+FROM golang:1.5 as builder
```

Comment:
Yeah, this wasn't a thing when I started, but I definitely approve of the "FROM AS" approach, heh. Much better than what was here before.